pyproj/_transformer.pyx

```diff
@@ -104,6 +104,14 @@ dy_dphi: List[float]
     Partial derivative of coordinate.
 """
 
+cdef PJ_DIRECTION get_pj_direction(object direction) except *:
+    try:
```

Comment:
Mind making a note here that this is for performance and reference this PR (https://github.com/pyproj4/pyproj/pull/1205)? This is for a future programmer that wonders why it is done this way when the code could have been simpler.

Reply:
Yep, good suggestion!

Reply:
Thanks!